packages/config/config/devices/0x0063/26931_zw4006.json

```diff
@@ -242,7 +241,8 @@
 					"label": "27 mins",
 					"value": 110
 				}
-			]
+			],
+			"description": "Steps of 15 seconds for value 5 through 109"
```

Comment:
Please put the description below the label.

Any chance you've measured some of these? 😅
If I assume 5 is 60s, 6 is 75s, etc. then 108 is 27 minutes, 109 is 27m15s and 110 is 27m again? Seems odd.

Reply:
So ... they come out at 15 seconds after the first two.

1 -> 10 seconds
2 -> 20 seconds
3 -> 30 seconds

and 15 seconds from there. So *actually* 110 is 27 minutes plus 15 seconds. (110*15/60-15=27.25)

It's accurate for values of 20, 8, and 9 at least. The description of [this one](https://products.z-wavealliance.org/products/2093/configs) at z-wavealliance.org shows it best - confirming how it actually works...

I'll change it to that wording "15 second intervals, starting at 60 seconds" like it is in that link.